pytest_rpc/__init__.py

```diff
@@ -14,21 +14,37 @@
 # ======================================================================================================================
 # Globals
 # ======================================================================================================================
-ENV_VARS = ['BUILD_URL',
-            'BUILD_NUMBER',
-            'RE_JOB_ACTION',
-            'RE_JOB_IMAGE',
-            'RE_JOB_SCENARIO',
-            'RE_JOB_BRANCH',
-            'RPC_RELEASE',
-            'RPC_PRODUCT_RELEASE',
-            'OS_ARTIFACT_SHA',
-            'PYTHON_ARTIFACT_SHA',
-            'APT_ARTIFACT_SHA',
-            'REPO_URL',
-            'JOB_NAME',
-            'MOLECULE_TEST_REPO',
-            'MOLECULE_SCENARIO_NAME']
+ASC_ENV_VARS = ['BUILD_URL',
+                'BUILD_NUMBER',
+                'RE_JOB_ACTION',
+                'RE_JOB_IMAGE',
+                'RE_JOB_SCENARIO',
+                'RE_JOB_BRANCH',
+                'RPC_RELEASE',
+                'RPC_PRODUCT_RELEASE',
+                'OS_ARTIFACT_SHA',
+                'PYTHON_ARTIFACT_SHA',
+                'APT_ARTIFACT_SHA',
+                'REPO_URL',
+                'JOB_NAME',
+                'MOLECULE_TEST_REPO',
+                'MOLECULE_SCENARIO_NAME']
+MK8S_ENV_VARS = ['BUILD_URL',
```

Comment:
We should also have the following variables:

```
BRANCH_NAME
CHANGE_AUTHOR_DISPLAY_NAME
CHANGE_AUTHOR
CHANGE_BRANCH
CHANGE_FORK
CHANGE_ID
CHANGE_TARGET
CHANGE_TITLE
CHANGE_URL
JOB_URL
NODE_LABELS
NODE_NAME
PWD
STAGE_NAME
```

The STAGE_NAME will be important to distinguish between our lightweight QC test suite and our full test suite.